exp-player/addon/templates/components/exp-audioplayer.hbs

```diff
@@ -1,13 +1,26 @@
 <div class="row">
+  {{#unless fullControls}}
     <div class="col-md-2">
-        {{#unless fullControls}}
-            <button {{action 'loopSound'}} class="player-audio-replay-widget">
-                <br>Replay
-            </button>
-        {{/unless}}
+      <button {{action 'loopSound'}} class="player-audio-replay-widget">
+        <br>Replay
+      </button>
     </div>
+  {{/unless}}
 
     <div class="col-md-10">
+      {{#if title}}
+        <h3>{{title}}</h3>
+      {{/if}}
+      {{#if titlePrompt}}
+        {{#with titlePrompt as |prompt|}}
```

Comment:
Does this really need to be aliased?

If we are aliasing this, it kind of suggests to me that the original names are confusing, and that may be the better place to make changes.